lib/notification.php

```diff
@@ -14,6 +14,15 @@ public static function sendMail($path){
 			return;
 		}
 		$email = \OCP\Config::getUserValue(\OCP\User::getUser(), 'settings', 'email', '');
+		if (\OCP\App::isEnabled(user_ldap)){
```

Comment:
I think it depends on what's the correct value in database. It is not helpful (in my opinion) if the mailheader looks like 
D14AB3F9-08F1-43AE-B83D-1D678A8A5416 <mail@myhost.com>
should be
My Name <mail@myhost.com>